### PR TITLE
Add `release:latest` to Search docs

### PR DESCRIPTION
**Pull request by MimiDumpling**
**BEFORE**:
image

**AFTER**:
image

### CODE REVIEW

**Comment by adhiraj** [comment]
@mimi `latest` is just a value they can use for the `release` keyword, so it should be within the `release` section. We can say "Exact match on the version of a release, or `latest` to pick the latest release". Same applies to the `first-release` keyword

**Comment by MimiDumpling** [comment]
> @mimi `latest` is just a value they can use for the `release` keyword, so it should be within the `release` section. We can say "Exact match on the version of a release, or `latest` to pick the latest release". Same applies to the `first-release` keyword

Hi @adhiraj, `first-release` is under Issue Properties and `release` is under Event Properties. Looks like we should keep them separate to differentiate? So, I made your suggested changes to `release`, but kept `first-release` the same. Let me know if there's a better way to define the properties. 😸

**Comment by adhiraj** [comment]
latest can be used with first-release too, to denote the latest release

**Comment by MimiDumpling** [comment]
> `latest` value can be used with `first-release` keyword too, to denote the latest release

Nice! Thanks for the clarification. Made the changes.

File: src/collections/_documentation/workflow/search.md [hunk 1]
```diff
@@ -197,8 +197,7 @@ Below is a list of Event-level tokens reserved and known to Sentry:
 
 : Restrict results to events tagged with the given release.
 
-  Exact match on the version of a release.
-
+  Exact match on the version of a release, or `release:latest` to pick the most recent release.
```
Comment (adhiraj) [inline src/collections/_documentation/workflow/search.md:200]: this should be `first-release:latest`

Comment (adhiraj) [reply src/collections/_documentation/workflow/search.md:200]: sorry I mean the one above should be

Comment (MimiDumpling) [reply src/collections/_documentation/workflow/search.md:200]: 🤦 My bad. Nice catch, Adi

**Pull request merged**